sklearn/plot/_heatmap.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+
+
+def plot_heatmap(values, xlabel="", ylabel="", xticklabels=None,
+                 yticklabels=None, cmap=None, vmin=None, vmax=None, ax=None,
+                 fmt="{:.2f}", xtickrotation=45, norm=None):
+    """Plot a matrix as heatmap with explicit numbers.
+
+    Parameters
+    ----------
+    values : ndarray
+        Two-dimensional array to visualize.
+
+    xlabel : string, default=""
+        Label for the x-axis.
+
+    ylabel : string, default=""
+        Label for the y-axis.
+
+    xticklabels : list of string or None, default=None
+        Tick labels for the x-axis.
+
+    yticklabels : list of string or None, default=None
+        Tick labels for the y-axis
+
+    cmap : string or colormap
+        Matpotlib colormap to use.
+
+    vmin : int, float or None
+        Minimum clipping value.
+
+    vmax : int, float or None
+        Maximum clipping value.
+
+    ax : axes object or None
+        Matplotlib axes object to plot into. If None, the current axes are
+        used.
+
+    fmt : string, default="{:.2f}"
+        Format string to convert value to text.
+
+    xtickrotation : float, default=45
+        Rotation of the xticklabels.
+
+    norm : matplotlib normalizer
+        Normalizer passed to pcolor
+    """
+    import matplotlib.pyplot as plt
+    if ax is None:
+        ax = plt.gca()
+    img = ax.pcolor(values, cmap=cmap, vmin=None, vmax=None, norm=norm)
+    # this will allow us to access the pixel values:
+    img.update_scalarmappable()
+    ax.set_xlabel(xlabel)
+    ax.set_ylabel(ylabel)
+
+    ax.set_xlim(0, values.shape[1])
```

Comment:
Yeah it's not necessary on 2.0.0rc2, so we only have to support it for like 8 1/2 more years or something (requirements are two ubuntu LTS releases ago)